src/main/java/com/aca/homework/week21/button/state/ButtonOffState.java

```diff
@@ -0,0 +1,20 @@
+package com.aca.homework.week21.button.state;
+
+public class ButtonOffState implements ButtonState {
+
+    @Override
+    public boolean isEnabled() {
+        return false;
+    }
+
+    @Override
+    public ColorType color() {
+        return ColorType.RED;
+    }
+
+    @Override
+    public ButtonState nextState() {
+        return new ButtonOnState();
```

Comment:
please do not create unnecessary objects every time. instead, please use singleton pattern